settings.py

```diff
@@ -69,15 +73,22 @@ def invertColor(self):
         self.bgColor = getInvertedRGB(self.bgColor)
         self.bulletColor = getInvertedRGB(self.bulletColor)
 
+    def speedVariable(self):
```

Comment:
How about consider using **Enum** ? Python support for enum since python 3.4.
https://docs.python.org/3.4/library/enum.html